### ISSUE TITLE
consider using Tika Python?

### ISSUE THREAD

**Issue by chrismattmann** [KBNLresearch/tikadetect-tree#1]
Tika python already downloads the network server and manages its execution

**Comment by bitsgalore** [comment]
@chrismattmann yes, I'm aware of that, and I wouldn't recommend my own script as a general-purpose solution. Main reason I created the script was to do a really basic and minimal PoC of how Tika detection might be used in one of our pre-ingest workflows (which are largely Bash based).
